packages/theme/src/lib/light-theme.ts

```diff
@@ -972,5 +980,45 @@ export const lightTheme = {
         },
       },
     },
+    SmileButtons: {
```

Comment:
Since we're not exporting `smilebuttons` it's probably better to create this as a slot under `feedbackform`

Reply:
Done